packages/ui/components/unpublished-entity/UnpublishedEntity.tsx

```diff
@@ -22,13 +22,8 @@ export function UnpublishedEntity(props: UnpublishedEntityProps) {
         }
         headline={t("team_is_unpublished", {
           team: props.name,
-          defaultValue: `${props.name} is unpublished`,
-        })}
-        description={t(`${props.orgSlug ? "org" : "team"}_is_unpublished_description`, {
-          defaultValue: `This ${
-            props.orgSlug ? "organization" : "team"
-          } link is currently not available. Please contact the organization owner or ask them to publish it.`,
         })}
+        description={t(`${props.orgSlug ? "org" : "team"}_is_unpublished_description`)}
```

Comment:
Removed default values introduced recently as 99% of the extra page size came from logo not i18n